tests/hooks/event-conversion-utils.spec.ts

```diff
@@ -108,6 +114,59 @@ describe('filterByWhitelist', () => {
   })
 })
 
+describe('getRegion', () => {
+  it('defaults in TBD if there is no region', () => {
+    const { tbd } = WORLD_REGIONS
+    const noRegionEvent = new FakeRecord({
+      name: 'Fake Conference'
+    })
+    expect(getRegion(noRegionEvent)).toBe(tbd)
+  })
+
+  it('gets the region from the record', () => {
+    const { americas } = WORLD_REGIONS
+    const fakeEvent = new FakeRecord({
+      name: 'Fake Conference',
+      region: americas
+    })
+    expect(getRegion(fakeEvent)).toBe(americas)
+  })
+
+  it('gets the region from the record even if it is not recognized by qiskit', () => {
+    const unknownRegionEvent = new FakeRecord({
+      name: 'Fake Conference',
+      region: 'Lemuria'
+    })
+    expect(getRegion(unknownRegionEvent)).toBe('Lemuria')
+  })
+})
+
+describe('getLocation', () => {
+  it('defaults in TBD if there is no location, no region', () => {
+    const nowhereEvent = new FakeRecord({
+      name: 'Nowhere Conference'
+    })
+    expect(getLocation(nowhereEvent)).toBe('TBD')
+  })
+
+  it('defaults in region if there is no location', () => {
+    const { americas } = WORLD_REGIONS
+    const noLocationEvent = new FakeRecord({
+      name: 'Fake Conference',
+      region: americas
+    })
+    expect(getLocation(noLocationEvent)).toBe(getRegion(noLocationEvent))
```

Comment:
It is a good practice to use a raw value in the matcher function instead of a function. Could you replace `getRegion(noLocationEvent)` with `americas`?

Reply:
I'll review the code, I think I have another instance of this out there.